utils/getLiteralCategoryList.ts

```diff
@@ -0,0 +1 @@
+export const getLiteralCategoryList = (data: Response<CategoryListProps>) => data.data.map(({ name }) => name);
```

Comment:
```suggestion
export const getLiteralCategoryList = (data: Response<CategoryListProps>) => data.map(({ name }) => name);
```

이럼 너무 좋을듯! 나머지 너무 깔끔하다 체고